src/ui/InsertWordUI.js

```diff
@@ -70,14 +70,17 @@ export default class InsertWordUI {
 
       this.mediaRecorder.stop();
     });
+
+    // 5. addEventListener
+    this.clearBtn.addEventListener("click", () => this.clear());
   }
 
   createWord() {
```

Comment:
should check the following conditions:
1. `word` input field should never be empty
2. the following 3 fields should have a least one fields that is not empty

If the above conditions are not satisfied, you should not create the word, nor save the word, but instead return `null`